### ISSUE TITLE
Tippy.js library not working with fancyBox

### ISSUE THREAD

**Issue by acwolff** [fancyapps/fancybox#2068]
I use in my [FancyBox skin](https://jalbum.net/en/skins/skin/FancyBox) the fancyBox library with the [Tippy.js library](https://atomiks.github.io/tippyjs/) for tool-tips, see [here](https://andrewolff.jalbum.net/FancyBoxWeb/) for an example.

The first window is a fancyBox modal window. You see there no tooltip at all for the close button. If I remove Tippy, you see the windows tool-tip “close”.
Click the close button to open the thumbnails page, you see that the tippy tool-tips are correctly working there.
Next click on a thumbnail to open the fancyBox light-box . Now you see for the tool-bar buttons the windows tool-tips, not the tippy tool-tips.

To investigate the problem I made a simple web page with your default example, where I added a title attribute to your thumbnails and added the tippy.min.js library, the result is [here](https://www.andrewolff.nl/fancybox-v3-4-basics-tippy).

You see that the tool-tips work correctly for the thumbnails, but if I open the fancyBox light-box  I don't see the tippy tool-tips for the 5 buttons on that page.

Do you have any idea  how to solve this problem?

**Comment by acwolff** [comment]
Yes, after the next release I will spent some time to get myself familiar with codepen!

Thanks again for your help, I implemented your solution this [this album](https://andrewolff.jalbum.net/FancyBoxWeb/). It works now fine in the light-box; only in modal windows, like the first window you see in my album, it is still not working.

I open that modal window after fancyBox has been initialized with this code:

	$.fancybox.open({
	src  : '#infoWindow',
	opts : {
		onInit: function(instance) {
			tippy('[title]', {
      			appendTo : instance.$refs.container[0],
      			zIndex : 99999
    		       });
		},
		modal: true
		}
    });

**Comment by fancyapps** [comment]
The problem is the same as the previous one, because `onInit` callback is executed right after interface is build, but before any content is loaded yet. Simply replace `onInit` with `afterShow`.

**Comment by fancyapps** [comment]
btw, I see that you have modified some CSS, but I think it would look better if you would add something like this:

```CSS
 .fancybox-is-closing {
  	opacity: 0;
  }
```

**Comment by acwolff** [comment]
Thanks again! I replaced onInit by afterShow and now is the first modal window OK.

I added also your  code
.fancybox-is-closing {
  	opacity: 0;
  }
code, but I can't see any difference!
What should I see?

**Comment by fancyapps** [comment]
The difference is that you can now see zoom-out animation when closing image, because container is fading out too.

**Comment by acwolff** [comment]
The correcty code for a  modal window is:

	$.fancybox.open({
	src  : '#infoWindow',
	opts : {
		onInit: function(instance) {
			tippy('[title]', {
      			// appendTo : instance.$refs.container[0],
      			zIndex : 99999
    		       });
		},
		modal: true
		}
    });

Yoy have to remove AppendTo, otherwise it does not work if you open the modaL WINDOIW FOR THE 2ND TIME.

**Comment by fancyapps** [comment]
Well, I am not expert of Tipsy, I think I saw it yesterday for the first time.

**Comment by acwolff** [comment]
Well it is not required on a PC, but on a touch screen device it replaces the mouse over tooltips of a PC. so the viewer gets an explanation of the functions of the buttons.

To see it s effect, open [this album](https://andrewolff.jalbum.net/FancyBoxWeb/) on a mobile device and put your finger some time on a button or image.
